From: David Herrmann <dh.herrmann@gmail.com>
Date: Thu, 12 Mar 2015 17:27:31 +0100
Subject: [PATCH] kdbus: samples/kdbus: add -lrt

On older systems -lrt is needed for clock_gettime(). Add it to
HOSTLOADLIBES of kdbus-workers so it builds fine on those systems.

Reported-by: Sasha Levin <sasha.levin@oracle.com>
Signed-off-by: David Herrmann <dh.herrmann@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 samples/kdbus/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/samples/kdbus/Makefile b/samples/kdbus/Makefile
index d009025369f4..eee9b9aed632 100644
--- a/samples/kdbus/Makefile
+++ b/samples/kdbus/Makefile
@@ -8,3 +8,4 @@ always := $(hostprogs-y)
 HOSTCFLAGS_kdbus-workers.o +=		\
 	-I$(objtree)/usr/include/	\
 	-I$(objtree)/include/uapi/
+HOSTLOADLIBES_kdbus-workers := -lrt
